app/src/main/AndroidManifest.xml

```diff
@@ -20,7 +24,9 @@
             android:name=".activities.ProjectInformationActivity"
             android:parentActivityName=".activities.ProjectsListActivity" />
         <activity android:name=".SignInActivity" />
-        <activity android:name=".Form" />
+        <activity
+            android:name=".Form"
+            android:screenOrientation="portrait" />
```

Comment:
Error from Cirrus :
Expecting `android:screenOrientation="unspecified"` or `"fullSensor"` for this activity so the user can use the application in any orientation and provide a great experience on Chrome OS device

To me it seems important, exept if we want to keep the app in only portrait mode

Reply:
Thanks for your review.
I force the portrait orientation as other orientation provide a poor user experience, and avoid some visual glitch, slowdowns when changing accidentally orientation.